lib/ssl/test/ssl_api_SUITE.erl

```diff
@@ -3205,6 +3205,8 @@ options_identity(_Config) ->  %% psk_identity  srp_identity and user_lookup_fun
         [], client),
     ?OK(#{psk_identity := <<"foobar">>, srp_identity := undefined, user_lookup_fun := undefined},
         [{psk_identity, "foobar"}], server),
+    ?OK(#{psk_identity := <<"">>, srp_identity := undefined, user_lookup_fun := undefined},
+        [{psk_identity, ""}], server),
```

Comment:
We should use client in test as I think we do not want to promote this usage in servers although it might not be forbidden by old PSK spec.